index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Luis Laverde's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navbar">
+    <div><img src="./imgs/my-logo/my-logo.png" /></div>
+    <div class="hamburguer-menu"><img src="./imgs/hamburguer-menu/hamburguer-menu-icon-enabled.png" /></div>
+  </nav>
+  <section class="name">
+    <header class="container">
+      <h1>
+        I'm Luis.
+        <br>
+        Glad to see
+        <br>
+        you!
+      </h1>
+      <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2>LET'S CONNECT</h2>
+      <ul>
+        <li><img src="./imgs/icons/Twitter.svg" /></li>
+        <li><img src="./imgs/icons/Linkedin.svg" /></li>
+        <li><img src="./imgs/icons/Medium.svg" /></li>
+        <li><img src="./imgs/icons/Github.svg" /></li>
+        <li><img src="./imgs/icons/Angelist.svg" /></li>
+      </ul>
```

Comment:
- Great job for using `ul` and `li` for your social icons 👍 I think it would be better to warp each icon with a link (`a` tag). the `a` tag Apart from being semantically correct, it will let you connect your social networks using href.

Reply:
- Wrap each <img> tag of social media icon with <a> tag for semantic reasons.
- Fill each href attribute with the real info of each social media.